operator/controller/src/main/java/io/apicurio/registry/operator/utils/Utils.java

```diff
@@ -8,4 +14,42 @@ private Utils() {
     public static boolean isBlank(String value) {
         return value == null || value.isBlank();
     }
+
+    /**
+     * Merge source map into target map, NOT overriding any entries with the same key.
+     *
+     * @param target must not be null
+     */
+    public static <V> void mergeNotOverride(Map<String, V> target, Map<String, V> source) {
+        requireNonNull(target);
+        if (source != null) {
+            source.forEach(target::putIfAbsent);
+        }
+    }
+
+    /**
+     * Merge source list into target list, NOT overriding any items based on the equality of the extracted
+     * key.
+     *
+     * @param target must not be null
+     */
+    public static <T, K> void mergeNotOverride(List<T> target, List<T> source, Function<T, K> extractKey) {
```

Comment:
I'm convinced this can be done in a simpler way, let me know if I should take a stab at it :+1:

Reply:
Can you create a followup PR for this please? PR does not have to be perfect, is should be correct. I'll be happy to review your PR.

Reply:
I'm ok with this, agreed.

Reply:
Follow up PR created: https://github.com/jsenko/apicurio-registry/pull/1055